samples/cpp/benchmark_app/.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Same - suggesting to remove this, this file is not necessary to put under source control. Also please remove unnecessary binary files (.suo, Browse.VC.db, slnx.sqlite)